app/controllers/application_controller.rb

```diff
@@ -3,7 +3,6 @@ class ApplicationController < ActionController::Base
   include Wor::Paginate
   include Pundit
   protect_from_forgery with: :null_session
-  before_action :authenticate_user!
```

Comment:
Instead of removing the before  action from the Application Controller, you should skip the before action in the controllers that does not need authentication